api/dump.go

```diff
@@ -157,10 +158,12 @@ func (s *service) prepareDumpOpts() *rpc.CriuOpts {
 	return &opts
 }
 
-func (s *service) runcDump(ctx context.Context, root, containerID string, opts *container.CriuOpts, state *task.ProcessState) error {
+func (s *service) runcDump(ctx context.Context, root, containerID string, pid int32, opts *container.CriuOpts, state *task.ProcessState) error {
```

Comment:
does this break regular runc checkpointing? e.g not sysbox?

Reply:
No, a few lines down is:

```
if pid != 0 {
	crPid = int(pid)
} else {
	crPid = runcContainer.Pid
}
```